src/aap_eda/api/views/user.py

```diff
@@ -241,6 +241,9 @@ class UserViewSet(
 
     def filter_queryset(self, qs):
         qs = visible_users(self.request.user, queryset=qs)
+        if not self.kwargs or "pk" not in self.kwargs:
+            # TODO: hardcode for now, will need to use settings.SYSTEM_USERNAME
+            qs = qs.exclude(username="_system")
```

Comment:
Can we add `settings.SYSTEM_USERNAME` and default that to `_system`?

Reply:
No. It causes some other problem that needs to be addressed post GA.